pyth-sdk-terra/README.md

```diff
@@ -0,0 +1,71 @@
+# Pyth SDK Terra
+
+This crate provides utilities for reading price feeds from the [pyth.network](https://pyth.network/) oracle on the Terra network.
+The crate includes a library for reading and using Pyth data feeds in Terra.
+
+## Usage
+
+### Read price
+ 
+For reading the price you just need to call `query_price` function within your contract.
+
+```rust
+// let price_id = Binary::from(b"xyz...");
+
+// Contract address is defaulted to the mainnet address, for testnet use `testnet` feature flag.
```

Comment:
is there a standard approach to getting contract addresses on terra? Do you know what other terra libraries do for testnet/mainnet configuration?

Reply:
I checked multiple terra contracts, none has contract address inside itself and they provided it in their website. I'll remove contract address the lib. I'll ask Reisen too.

Reply:
Agreed with Reisen to remove. Adding later is cheaper than removing 😅